packages/betterer/src/files/file-info.ts

```diff
@@ -5,9 +5,9 @@ import { getConfig } from '../config';
 export type BettererFileCodeInfo = LoggerCodeInfo;
 
 type BettererFileErrorMap = Record<string, Array<BettererFileCodeInfo> | null>;
-type BetterFileLoc = [number, number];
-type BettererFileLocs = Array<BetterFileLoc>;
-export type BettererFileInfoSerialised = Record<string, BettererFileLocs>;
+type BetterFileMark = [number, number, string?];
+type BettererFileMarks = Array<BetterFileMark>;
```

Comment:
This didn't actually have to be renamed... I just didn't think Loc would be fully representative of it. `Mark` is supposed to be like... a file marker? But that might not make sense either. 

Names are hard.